modules/vardictjava/main.nf

```diff
@@ -0,0 +1,43 @@
+process VARDICTJAVA {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::vardict-java=1.8.3" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/vardict-java:1.8.3--hdfd78af_0':
+        'quay.io/biocontainers/vardict-java:1.8.3--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(bam), path(regions_of_interest)
```

Comment:
I'm not sure the ROI should be included in this tuple, but rather handled as a separate file? (as it was before).
Is there a reason for this?

Reply:
Not really a reason for it, I changed them around, because this input was swapped. I'll split it into a seperate input :)